backend/src/server/routes/v1/secret-folder-router.ts

```diff
@@ -127,6 +127,71 @@ export const registerSecretFolderRouter = async (server: FastifyZodProvider) =>
     }
   });
 
+  server.route({
+    url: "/batch",
+    method: "PATCH",
+    config: {
+      rateLimit: secretsLimit
+    },
+    schema: {
+      description: "Update folders by batch",
+      security: [
+        {
+          bearerAuth: []
+        }
+      ],
+      body: z.object({
+        workspaceId: z.string().trim().describe(FOLDERS.UPDATE.workspaceId),
```

Comment:
Can we make the endpoint use project slugs? We are trying to move away from ID-based endpoints and instead move to slugs

So instead of `workspaceId` you would name it `projectSlug`

Here's an example route that uses project slugs instead of project ID's: https://github.com/Infisical/infisical/blob/b4a2123fa3812eb1bb9d73333865f4d96480d58c/backend/src/ee/routes/v1/dynamic-secret-router.ts#L158